CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.1.39
+
+Add [supplier](https://developer.ravelin.com/apis/v2/#postv2supplier) object
+
+# 0.1.38
+
+Add a include_rule_output option
```

Comment:
I think you forgot about changelog @byarr last time

Reply:
I just pleased I remembered to bump the version number!